vagrant/provisioning/roles/confluent-platform-install/tasks/main.yml

```diff
@@ -1,5 +1,9 @@
 ---
 # tasks file for confluent-install
+- name: Upgrade confluent platform
+  import_tasks: upgrade-confluent.yml
+  when: confluent_major_version is version('6.2', '<')
```

Comment:
This won't work after you've updated the confluent_major_version to 6.2 in the facts file. We need to check the version installed and run this if the versions differ. You can extract the version from this command: `rpm -qa | grep confluent-platform`

Reply:
Note that this will not return anything if confluent-platform isn't installed already.